src/main/fc/fc_rc.c

```diff
@@ -854,3 +855,34 @@ bool rcSmoothingInitializationComplete(void) {
     return (rxConfig()->rc_smoothing_type != RC_SMOOTHING_TYPE_FILTER) || rcSmoothingData.filterInitialized;
 }
 #endif // USE_RC_SMOOTHING_FILTER
+
+FAST_CODE float stickFeels(float rcCommand, int axis)
+{
+  static FAST_RAM_ZERO_INIT float lastRcCommandData[3];
+  float pterm_low, pterm_high, pterm, iterm_low, iterm_high, iterm, dterm_low, dterm_high, dterm;
+  float rcCommandPercent;
+  float rcCommandError;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'rcCommandError' can be reduced.](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5192242)